lib/paged_vertical_calendar.dart

```diff
@@ -407,3 +402,41 @@ typedef MonthBuilder = Widget Function(
 typedef DayBuilder = Widget Function(BuildContext context, DateTime date);
 
 typedef OnMonthLoaded = void Function(int year, int month);
+
+List<DateTime> listOfDatesInMonth(Month month, List<WeekDay> weekdaysToHide) {
+  final totalDays = month.daysInMonth;
+  final validDates = <DateTime>[];
+  for (int i = 1; i <= totalDays; i++) {
+    final date = DateTime(month.year, month.month, i);
+    if (!weekdaysToHide.contains(getWeekDayByDartDayValue(date.weekday))) {
+      validDates.add(date);
+    }
+  }
+  return (validDates);
+}
+
+int getIndexOfFirstValidDayInMonth(Month month, List<WeekDay> weekdaysToHide,
+    {required bool isSundayFirstDayOfWeek}) {
```

Comment:
A required, optional named parameter is a weird pattern, provide a default for `isSundayFirstDayOfWeek` or remove named brackets.